demos/interactive_face_detection_demo/interactive_face_detection.hpp

```diff
@@ -32,64 +28,70 @@ static const char plugin_message[] = "Plugin name. For example, CPU. If this par
 
 /// @brief Message for assigning face detection calculation to device
 static const char target_device_message[] = "Optional. Target device for Face Detection network (the list of available devices is shown below). " \
-"Default value is CPU. Use \"-d HETERO:<comma-separated_devices_list>\" format to specify HETERO plugin. " \
-"The demo will look for a suitable plugin for a specified device.";
+"Default value is CPU. The demo will look for a suitable plugin for a specified device.";
```

Comment:
Why did you change the text?

Reply:
Still a question

Reply:
Didn`t check this feature, so I changed the text
Now I check this, its work correctly
Returned text